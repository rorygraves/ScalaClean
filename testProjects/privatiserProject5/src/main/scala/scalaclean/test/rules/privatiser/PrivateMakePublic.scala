package x1.scalaclean.test.rules.privatiser.MakePublic {

  object AppPublic extends App {
    import x2.scalaclean.test.rules.privatiser.MakePublic.Other._
    println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    import x2.scalaclean.test.rules.privatiser.MakePublic.Other3.{ myVal => myValOn3 }
    println(myValOn3)
    x2.scalaclean.test.rules.privatiser.MakePublic.Access.access2()
    x2.scalaclean.test.rules.privatiser.MakePublic.Access.access3()
  }

}

package x2.scalaclean.test.rules.privatiser.MakePublic {

  object Other {
    val myVal = 1
    var myVar = 1
    def myDef = 1
    object myObj
    trait myTrait
    class myClass
  }

  object Other2 {
    val myVal = 1
    var myVar = 1
    def myDef = 1
    object myObj
    trait myTrait
    class myClass
  }

  object Other3 {
    val myVal = 1
    var myVar = 1
    def myDef = 1
    object myObj
    trait myTrait
    class myClass
  }

  object Access {

    def access2() = {
      import x2.scalaclean.test.rules.privatiser.MakePublic.Other2._
      println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    }

    def access3() = {
      import x2.scalaclean.test.rules.privatiser.MakePublic.Other3._
      println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    }

  }

}
