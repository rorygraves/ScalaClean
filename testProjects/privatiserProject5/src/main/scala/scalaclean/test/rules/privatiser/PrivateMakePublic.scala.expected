package x1.scalaclean.test.rules.privatiser.MakePublic {

  object AppPublic extends App {
    import x2.scalaclean.test.rules.privatiser.MakePublic.Other._
    println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    import x2.scalaclean.test.rules.privatiser.MakePublic.Other3.{ myVal => myValOn3 }
    println(myValOn3)
    x2.scalaclean.test.rules.privatiser.MakePublic.Access.access2()
    x2.scalaclean.test.rules.privatiser.MakePublic.Access.access3()
  }

}

package x2.scalaclean.test.rules.privatiser.MakePublic {

  object Other {
    val myVal = 1
    var myVar = 1
    def myDef = 1
    object myObj
    protected[this] trait myTrait
    class myClass
  }

  private[MakePublic] object Other2 {
    private[MakePublic] val myVal = 1
    private[MakePublic] var myVar = 1
    private[MakePublic] def myDef = 1
    private[MakePublic] object myObj
    protected[this] trait myTrait
    private[MakePublic] class myClass
  }

  object Other3 {
    val myVal = 1
    private[MakePublic] var myVar = 1
    private[MakePublic] def myDef = 1
    private[MakePublic] object myObj
    protected[this] trait myTrait
    private[MakePublic] class myClass
  }

  object Access {

    def access2() = {
      import x2.scalaclean.test.rules.privatiser.MakePublic.Other2._
      println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    }

    def access3() = {
      import x2.scalaclean.test.rules.privatiser.MakePublic.Other3._
      println(myVal + myVar + myDef + myObj.toString + (new myClass + (new myTrait {}).toString))
    }

  }

}
